sdk/eventhub/test-resources.json

```diff
@@ -87,6 +87,15 @@
         "tier": "Standard"
       }
     },
+    {
```

Comment:
Any chance that I can talk you into using the Event Hubs infrastructure to create a scope in your test setup and tear it down after?   This kind of resident state makes local development more annoying unless you're running the resource creation and cleanup after every run.

Same plea for storage.